setup.sh

```diff
@@ -66,6 +71,12 @@ apt-get install -y supervisor
 cp /usr/share/sonar_poller/config/sonar_poller.conf /etc/supervisor/conf.d/
 systemctl restart supervisor
 
+
+## Install vendor libraries needed for poller
+cd /usr/share/sonar_poller/
+composer install
```

Comment:
This is already on line 60, so shouldn't be needed here (and shouldn't be run as root)